### PR TITLE
Add and update the installation steps and files.

### PR DESCRIPTION
**Pull request by luxonauta**
- :memo: Update the installation steps.
- :art: Updates the screenshot based on the new theme version.
- :package: Added the theme installation file for Visual Studio 2022 - this file is created using the [Theme Converter for Visual Studio](https://github.com/microsoft/theme-converter-for-vs) tool.

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -1,44 +1,56 @@
-### [Visual Studio](https://www.visualstudio.com/)
```
Comment (zenorocha) [inline INSTALL.md:1]: This title is important and can't be removed, without that it will not render the website correctly.

File: INSTALL.md [hunk 2]
```diff
@@ -1,44 +1,56 @@
...
+### Downloading using Git
```
Comment (zenorocha) [inline INSTALL.md:1]: This should be a an H4 (`####`)

File: INSTALL.md [hunk 3]
```diff
@@ -1,44 +1,56 @@
...
-    git clone https://github.com/dracula/visual-studio
+The default branch is the most recent year of Visual Studio (2022) we support. If you need to use an older version like 2019, you will need to checkout that branch.
 
-The default branch is the latest year of Visual Studio that we support. If you need to use an older version such as 2012, you need to checkout that branch.
+```shell
+git checkout 2019
+```
 
-    git checkout 2012
+### Downloading manually
```
Comment (zenorocha) [inline INSTALL.md:15]: Same here

File: INSTALL.md [hunk 4]
```diff
@@ -1,44 +1,56 @@
...
+4. Click "Open";
+5. Select the `Dracula Theme` theme option to preview it;
+6. Click "Ok" to apply it;
+7. Ready! 🚀
+
+> If you followed the steps above and could not install it, please [file an issue on GitHub](https://github.com/dracula/visual-studio/issues/new) or browse [our Discord](https://draculatheme.com/discord-invite), where the community can help.
+
+### Visual Studio
+
+If you have questions about Visual Studio, the best place to look for help is on the [official Visual Studio product family documentation](https://docs.microsoft.com/en-gb/visualstudio/).
```
Comment (zenorocha) [inline INSTALL.md:56]: This whole section isn't needed and also doesn't follow the standard of other repos.

**Review by zenorocha — CHANGES_REQUESTED** [review]
The screenshot also needs to be updated - having a background with any color could become an issue when we update the website colors.

**Comment by zenorocha** [comment]
I noticed that the theme name on Visual Studio is `draculatheme`, can we change to `Dracula Theme`

image

**Comment by zenorocha** [comment]
Should the [dracula.vssettings](https://github.com/dracula/visual-studio/blob/master/dracula.vssettings) file be removed?

**Comment by luxonauta** [comment]
> Should the [dracula.vssettings](https://github.com/dracula/visual-studio/blob/master/dracula.vssettings) file be removed?

Zeno, I thought about removing it, but as I don't know if this method works on Mac OS, I left the file thinking about compatibility; for that reason, I just left the steps for installing on Mac OS the same.

**Comment by luxonauta** [comment]
> I noticed that the theme name on Visual Studio is `draculatheme`, can we change to `Dracula Theme`
> 
> image

:pencil2::white_check_mark: Fixed Zeno:

image

**Review by zenorocha — APPROVED** [review]

**Pull request merged**